### ISSUE TITLE
Don't use Gouraud shading on minimap

### ISSUE THREAD

**Issue by Spikeone** [Return-To-The-Roots/s25client#414]
Currently we use the block (I've forgotten the block id) Gouraud shading for map previews in the map selection screen. As we do not use those information anywhere else we might use an other way for displaying map previews.

**Comment by Flamefire** [comment]
Why? Minimaps look nice the way they are, don't they? Without those shading stuff the maps would not suggest mountains etc.

**Comment by Spikeone** [comment]
Well, it's not used ingame at all. So why should we use this and add it to a mapfile when it's only purpose is mappreview.

The preview could also use heights.

**Comment by Flamefire** [comment]
It is already there and required by the S2 map format... So for S2 compatible maps it is already there. But yes, we could ignore this and calculate it as e.g. in #415 it can be removed.

**Comment by Spikeone** [comment]
Well I was just playing around with my tool and always thought the bad preview was caused by not adding area information. Then I just added a player with a hex editor and the map worked perfectly. I agree that most editors generate shading but... I won't and I don't see any reason to use the block at all. If that referenced report is the short map format, then this could be part of as a comverter could be added in order tore move those information.

I don't think our format must be compatible with the original anymore.
